content/guides/concepts/dnslink.md

```diff
@@ -43,3 +51,24 @@ Will get you this block:
 ```
 /ipfs/QmYNQJoKGNHTpPxCBPh9KkDpaExgd2duMa3aF6ytMpHdao/media/
 ```
+
+### Publish via Subdomain
+
+It is possible to publish DNSLink using a special subdomain named `_dnslink`.
+
+Example: `docs.ipfs.io` may not not have a TXT record, but the page still loads
+because TXT record exists for `_dnslink.docs.ipfs.io`:
+
+```
+dig +noall +answer TXT _dnslink.docs.ipfs.io
```

Comment:
Please add a `$` or `>` at the start of this line to show it’s input at the prompt and not output!